PocketCoffea/parameters/lumi.py

```diff
@@ -1,3 +1,10 @@
+# Function to convert and round the integrated luminosity from picobarn to femtobarn
```

Comment:
Is this really necessary? Ok

Reply:
Maybe it's over-engineered, but it's just to keep track of the reason why we divide the luminosity by a factor 1000.